Introduce a Related Resources section for n-tier-content, adding some additional links (#4218)
MicrosoftDocs/architecture-center: docs/guide/architecture-styles/n-tier-content.md

Layers are a way to separate responsibilities and manage dependencies. Each layer has a specific responsibility. A higher layer can use services in a lower layer, but not the other way around.

> Tiers are physically separated, running on separate machines. A tier can call to another tier directly, or use [asynchronous messaging (message queue)](/azure/service-bus-messaging/service-bus-async-messaging). Although each layer might be hosted in its own tier, that's not required. Several layers might be hosted on the same tier. Physically separating the tiers improves scalability and resiliency, but also adds latency from the additional network communication.

Reviewer: ```suggestion
Tiers are physically separated, running on separate machines. A tier can call to another tier directly, or use [Asynchronous messaging patterns](/azure/service-bus-messaging/service-bus-async-messaging) through a message queue. Although each layer might be hosted in its own tier, that's not required. Several layers might be hosted on the same tier. Physically separating the tiers improves scalability and resiliency, but also adds latency from the additional network communication.
```

> [!NOTE]
> The layer labeled "Business Tier" in our reference diagram is a moniker to the business logic tier. Likewise, we also call the presentation tier the "Web Tier." In our example, this is a web application, though multi-tier architectures can be used for other topologies as well (like desktop apps).  Name your tiers what works best for your team to communicate the intent of that logical and/or physical tier in your application - you could even express that naming in resources you choose to represent that tier (e.g. vmss-appName-business-layer).

For more information about running N-tier applications on Azure:

- [N-tier application with Apache Cassandra][n-tier-linux]
- [Windows N-tier application on Azure with SQL Server][n-tier-windows-SQL]
- [Microsoft Learn module: Tour the N-tier architecture style](/training/modules/n-tier-architecture/)

> - [Azure Bastion](/azure/bastion/bastion-overview)

Author: Moved to the Related Resources section below.

> - Use virtual machine scale sets for [autoscaling][auto-scaling].

Reviewer 2: ```suggestion
- Use virtual machine scale sets for [autoscaling][autoscaling].
```

- Look for places in the architecture where you can use a managed service without significant refactoring. In particular, look at caching, messaging, storage, and databases.

- For higher security, place a network DMZ in front of the application. The DMZ includes network virtual appliances (NVAs) that implement security functionality such as firewalls and packet inspection. For more information, see [Network DMZ reference architecture][dmz].

> - For high availability, place two or more NVAs in an availability set, with an external load balancer to distribute Internet requests across the instances. For more information, see [Deploy highly available network virtual appliances][ha-nva] and this example scenario of [High availability and disaster recovery for IaaS apps](/azure/architecture/example-scenario/infrastructure/iaas-high-availability-disaster-recovery)

Reviewer: ```suggestion
- For high availability, place two or more NVAs in an availability set, with an external load balancer to distribute Internet requests across the instances. For more information, see [Deploy highly available network virtual appliances][ha-nva] and this example scenario of [High availability and disaster recovery for IaaS apps](/azure/architecture/example-scenario/infrastructure/iaas-high-availability-disaster-recovery).
```

- If you need higher availability than the Azure SLA for VMs provides, replicate the application across two regions and use Azure Traffic Manager for failover. For more information, see [Run Windows VMs in multiple regions][multiregion-windows] or [Run Linux VMs in multiple regions][multiregion-linux].

### Related Resources

- [N-tier application with Apache Cassandra][n-tier-linux]
- [Windows N-tier application on Azure with SQL Server][n-tier-windows-SQL]
- [Microsoft Learn module: Tour the N-tier architecture style](/training/modules/n-tier-architecture/)
- [Azure Bastion](/azure/bastion/bastion-overview)

> - More information on using [messaging in N-Tier architectures](https://docs.particular.net/architecture/azure/n-tier)

Reviewer: ```suggestion
- More information on messaging in an [N-tier architecture style on Azure](https://docs.particular.net/architecture/azure/n-tier)
```

> 

Reviewer: ```suggestion
```
